dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrDataModel.ts

```diff
@@ -54,8 +54,7 @@ export const sfdrDataModel = [
           {
             name: "referencedReports",
             label: "Referenced Reports",
-            description:
```

Comment:
This is because there are indeed two fields with the same name in the DD in each case. However, one the second (new one) means the relative production/consumption compared to all the others and is measured in percent. This is unfortunately not reflected by the name here. I will change the names corresponding to the relative ones in the sense of giving them a prefix "Relative"

Reply:
I took care of this and created a small new devtools PR

Reply:
ok